profiles/killbill/src/test/resources/killbill.properties

```diff
@@ -21,6 +21,8 @@ org.killbill.overdue.uri=overdue.xml
 
 org.killbill.payment.retry.days=8,8,8
 
+org.killbill.payment.plugin.timeout=8s
```

Comment:
I think this only impact the tests in the profile module. As far as I could tell, all tests that use this setting are in the slow group.

But yeah, it would be great to scope this to the test. I didn't want to set it lower since I was afraid we'd start getting timeouts in other tests that don't expect timeouts.